### DISCUSSION TITLE
How to prevent the resizing of Frames based of buttons inside

### DISCUSSION THREAD

**Discussion by ThomasVenter** [TomSchimansky/CustomTkinter#1404] [Q&A]
Good day,

I am a new user and cant seem to find the method to forbib frame resizing. Ive tried dynamic_resizing=False and .resizable(False, False) but neither seem to work!

**Comment by avalon60** [comment]
Hi Thomas. 
Try this:

```
root = CTk()
root.resizable(False, False)
```

Should also work with CTkToplevel.

**Comment by VasigaranAndAngel** [comment]
> Good day,
> 
> I am a new user and cant seem to find the method to forbib frame resizing. Ive tried dynamic_resizing=False and .resizable(False, False) but neither seem to work!

did you try `.pack_propagate(False)`
